media-api/app/lib/ImageResponse.scala

```diff
@@ -230,8 +231,18 @@ object ImageResponse extends EditsResponse {
     (__ \ "usages").write[UsagesEntity]
       .contramap(usagesEntity(id, _: List[Usage])) ~
     (__ \ "collections").write[List[EmbeddedEntity[Collection]]]
-      .contramap((collections: List[Collection]) => collections.map(c => collectionsEntity(id, c)))
-  )(unlift(Image.unapply))
+      .contramap((collections: List[Collection]) => collections.map(c => collectionsEntity(id, c))) ~
+    (__ \ "collectionGroups").write[List[CollectionGroup]]
```

Comment:
Not sure this is going to be necessary if we try to show a hint of the top-level collection for each leaf collection (rather than grouping per top-level collection), something @tsop14 is looking into now.